src/main/scala/apps/harrisCornerDetectionHalideRewrite.scala

```diff
@@ -83,20 +84,20 @@ object harrisCornerDetectionHalideRewrite {
   }
 
   def isAppliedPair: Strategy[Rise] =
-    function(function(isEqualTo(rise.core.DSL.pair)))
+    function(function(isEqualTo(rise.core.primitives.pair.primitive)))
```

Comment:
is this equivalent to `isEqualToUntyped(pair())`?

Reply:
Yes.